.github/workflows/ci.yml

```diff
@@ -19,17 +19,17 @@ jobs:
       matrix:
         rust: ["1.59.0", "stable"]
     steps:
-      - uses: hecrj/setup-rust-action@967aec96c6a27a0ce15c1dac3aaba332d60565e2
+      - uses: hecrj/setup-rust-action@50a120e4d34903c2c1383dec0e9b1d349a9cc2b1
```

Comment:
Updated to [v1.4.1](https://github.com/hecrj/setup-rust-action/releases/tag/v1.4.1)